PEAR.php

```diff
@@ -508,7 +508,7 @@ function delExpect($error_code)
      * @see PEAR::setErrorHandling
      * @since PHP 4.0.5
      */
-    function &raiseError($message = null,
+    static function raiseError($message = null,
```

Comment:
This ones concerns me a bit -- if you dig into this function, it's actually designed to work in static *or* non-static usages. And there are several non-static usages in, e.g., `Net_Socket` and `Mail_mimeDecode`. We need to be careful that we're not just trading one set of errors for another.